http/headers/set-cookie.json

```
@@ -497,49 +511,73 @@
-                "firefox_android": {
-                  "version_added": "79",
-                  "flags": [
-                    {
-                      "type": "preference",
-                      "name": "network.cookie.sameSite.schemeful",
-                      "value_to_set": "true"
-                    }
-                  ]
-                },
+                "edge": [
+                  {
+                    "version_added": "89"
+                  },
+                  {
+                    "version_added": "86",
+                    "version_removed": "89",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": "#schemeful-same-site",
+                        "value_to_set": "Enabled"
+                      }
+                    ]
+                  }
+                ],
+                "firefox": [
+                  {
+                    "version_added": "96"
+                  },
+                  {
+                    "version_added": "79",
+                    "version_removed": "96",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": "network.cookie.sameSite.schemeful",
+                        "value_to_set": "true"
+                      }
+                    ]
+                  }
+                ],
+                "firefox_android": [
+                  {
+                    "version_added": "96"
+                  },
+                  {
+                    "version_added": "79",
+                    "version_removed": "96",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": "network.cookie.sameSite.schemeful",
+                        "value_to_set": "true"
+                      }
+                    ]
+                  }
+                ],
                 "ie": {
                   "version_added": false
                 },
-                "opera": {
-                  "version_added": "72",
-                  "flags": [
-                    {
-                      "type": "preference",
-                      "name": "#schemeful-same-site",
-                      "value_to_set": "Enabled"
-                    }
-                  ]
-                },
+                "opera": [
+                  {
+                    "version_added": "75"
+                  },
+                  {
+                    "version_added": "72",
+                    "version_removed": "75",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": "#schemeful-same-site",
+                        "value_to_set": "Enabled"
+                      }
+                    ]
+                  }
+                ],
                 "opera_android": {
                   "version_added": false
```

Add Opera on Android version for Blink 89

```suggestion
                  "version_added": ""63
```